### PR TITLE
create header content

### PR DESCRIPTION
**Pull request by Thieumsc**

### CODE REVIEW

File: public/assets/css/header.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    opacity: 0,5;
+    background-color: #0098FF;
+}
+
+body > header > nav > ul > li:last-child {
+    margin-left: 23.3px;
+}
+
+li {
+    margin: 0;
```
Comment (jojo957) [inline public/assets/css/header.css:15]:
```suggestion
   margin: 10px;
```

Comment (jojo957) [inline public/assets/css/header.css]: il faudrait supprimer cette ligne

File: public/assets/css/header.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    padding: 0;
```
Comment (jojo957) [inline public/assets/css/header.css]: delete this line.
this line should be in reset.css file

File: public/assets/css/header.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
+body > header > nav > ul > li:first-child {
+    width: 64px;
+    height: 62px;
+    border: 0 solid #0098FF;
```
Comment (mineirabrasil) [inline public/assets/css/header.css]: you can delete this line

Comment (jojo957) [inline public/assets/css/header.css]: why border: 0 ?
i think you must delete this line

File: public/assets/css/header.css [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    border-radius: 50%;
+    opacity: 0,5;
```
Comment (mineirabrasil) [inline public/assets/css/header.css]: you can delete this line

File: public/assets/css/header.css [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    background-color: #0098FF;
```
Comment (mineirabrasil) [inline public/assets/css/header.css]: you can replacethis line with
```suggestion
    background: rgb(128, 194, 233);
```

File: public/assets/css/header.css [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+body > header > nav > ul > li:last-child {
+    margin-left: 23.3px;
+}
+
+li {
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
```
Comment (jojo957) [inline public/assets/css/header.css]: delete this line.
this line should be in reset.css file

File: public/assets/css/header.css [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    display: inline-block;
+    font-family: Muli, Arial, 'sans-serif';
```
Comment (jojo957) [inline public/assets/css/header.css]: font-family: Anisse 
please

File: public/assets/css/header.css [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+    font-weight: 400;
+}
+
+a {
+    text-decoration: none;
```
Comment (jojo957) [inline public/assets/css/header.css]: delete this line.
this line should be in reset.css file

File: public/assets/css/header.css [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+    color: #000;
+    opacity: 0.7;
```
Comment (mineirabrasil) [inline public/assets/css/header.css]: you can delete this line

File: public/assets/css/header.css [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+    }
```
Comment (mineirabrasil) [inline public/assets/css/header.css]: you can add : 
```suggestion
    cursor : pointer;
    }
```

File: public/assets/css/header.css [hunk 11]
```diff
@@ -0,0 +1,27 @@
+body > header > nav > ul > li:first-child {
+    width: 64px;
+    height: 62px;
```
Comment (jojo957) [inline public/assets/css/header.css]: why width : 64px and height: 62px ?
i think width and height should be in the same size (width: 64px height: 64px)

File: public/assets/css/header.css [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+    border: 0 solid #0098FF;
+    border-radius: 50%;
+    opacity: 0,5;
+    background-color: #0098FF;
+}
+
+body > header > nav > ul > li:last-child {
+    margin-left: 23.3px;
+}
```
Comment (jojo957) [inline public/assets/css/header.css:6]: margin-left: 23.3px ?
delete this

File: public/assets/css/header.css (hunk 2 above)
Comment (mineirabrasil) [inline public/assets/css/header.css]: you can delete this line  since it already exist in reset.css

File: public/assets/css/header.css (hunk 6 above)
Comment (mineirabrasil) [inline public/assets/css/header.css]: you can delete this line  since it already exist in reset.css

File: public/assets/css/header.css (hunk 8 above)
Comment (mineirabrasil) [inline public/assets/css/header.css]: you can delete this line  since it already exist in reset.css

File: public/assets/css/header.css (hunk 9 above)
Comment (jojo957) [inline public/assets/css/header.css]: delete this line 
i think you should just change the color

**Review by jojo957 — CHANGES_REQUESTED** [review]

**Review by jojo957 — APPROVED** [review]

**Review by mineirabrasil — APPROVED** [review]

**Pull request merged**